download/upgradeRecipe/upgradeRecipe8.adoc

```diff
@@ -50,4 +50,176 @@ Any backwards incompatible changes are annotated with a [.label.label-danger.lab
 Due to the limited usage of OSGi and the maintenance burden it brings,
 the OptaPlanner jars in the 8.x series no longer include OSGi metadata in their `META-INF/MANIFEST.MF` file.
 
+[.upgrade-recipe-major]
+=== `filterClassList` replaced by a single filterClass
 
+This change applies to a configuration of `EntitySelector`, `ValueSelector` and `MoveSelector` in both
+the configuration API and the solver configuration XML.
+
+==== Reason:
```

Comment:
Including the motivation, as a second paragraph is really good.
But until now we typically didn't split it off in a separate subsection with a "Reason:" header. For consistency I'd leave it out, or apply it on all upgrade recipe texts?

Reply:
I was considering this to be a potential improvement of the upgrade recipe starting with version 8. And I am fine with adding the subsection into the other existing recipes in upgradeRecipe8. 

Adding it retrospectively to all upgrade recipes, in my opinion, does not make much sense as these have been already  published for a while. Moreover, can we reconstruct the motivation for each one of them?